product/opni/components/Cluster.vue

```diff
@@ -73,7 +87,98 @@ export default {
         this.clusterCountInterval = null;
         this.newClusterFound = true;
       }
-    }
+    },
+
+    async renderInstallCommand() {
+      if (!this.capability || !this.token) {
+        return;
+      }
+      const cmd = await getCapabilityInstaller(this.capability, this.token, this.pin);
+      const regex = /\%({.*?\})\%/g;
+      const inputArgs = [];
+
+      let match;
+
+      let parsedCmd = cmd;
+
+      while ((match = regex.exec(cmd)) !== null) {
+        const json = JSON.parse(match[1]);
+        let label;
+
+        switch (json.kind) {
+        case 'input':
+          label = json.input.label;
+          break;
+        case 'select':
+          label = json.select.label;
+          break;
+        case 'toggle':
+          label = json.toggle.label;
+          break;
+        }
+        let value;
+        let required = false;
+        let omitEmpty = false;
+        let format = '{{ value }}';
+
+        for (const option of json.options) {
+          switch (option.name) {
+          case 'default':
+            if (option.value === 'true') {
```

Comment:
By any chance can you change that json in the backend so it's a boolean type instead of a string?